### PR TITLE
Removed all css that caused build warnings

### LINKED ISSUE #907: Remove build warnings in example app
**Bug**

Currently the example app has css build warnings. These should be removed so the app can build successfully without warnings.

### PR DESCRIPTION
**Pull request by samenz**
**Type:** Bug

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**
Removes css that were causing build warnings. 

Pending fixing up tests

Resolves #907

### CODE REVIEW

File: src/slide-pane/index.ts [hunk 1]
```diff
@@ -305,7 +305,14 @@ export class SlidePane extends I18nMixin(ThemedMixin(WidgetBase))<SlidePanePrope
 											},
 											[
 												closeText,
-												w(Icon, { type: 'closeIcon', theme, classes })
+												w(Icon, {
+													type: 'closeIcon',
+													theme,
+													classes,
+													extraClasses: {
```
Comment (samenz) [inline src/slide-pane/index.ts]: If there is a better way to add additional classes to a widget please let me know.

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-8dtczhubr.now.sh

File: src/slide-pane/index.ts [hunk 2]
```diff
@@ -305,7 +305,14 @@ export class SlidePane extends I18nMixin(ThemedMixin(WidgetBase))<SlidePanePrope
 											},
 											[
 												closeText,
-												w(Icon, { type: 'closeIcon', theme, classes })
+												w(Icon, {
```
Comment (tomdye) [inline src/slide-pane/index.ts]: i'd argue that this sizing should either be a `size` property on the `Icon` widget or simply applied to a wrapping `span` instead. Would be the simplest way I believe.

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-266y9xgbl.now.sh
* Tests Deployment: https://dojowidgets-j2ci17hwt.now.sh

File: src/slide-pane/index.ts (hunk 2 above)
Comment (samenz) [reply src/slide-pane/index.ts]: Cool I wrapped the icon in a span and put the styling on that

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-ba0gr8v0y.now.sh
* Tests Deployment: https://dojowidgets-hjr675o5j.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-djv7nw1lm.now.sh
* Tests Deployment: https://dojowidgets-9hxm1dg8u.now.sh

**Pull request merged**